src/core/dbus.c

```diff
@@ -1166,15 +1167,33 @@ int bus_foreach_bus(
                 RET_GATHER(r, send_message(b, userdata));
         }
 
-        /* Send to API bus, but only if somebody is subscribed */
+        /* Send to API bus, if have subscribers and if we should honor them */
```

Comment:
The meaning of this comment is wrong? "Send to API bus, unless we should honor subscribers and there's none"?

Reply:
No, if we honor subscribers then we only send it when the list is not empty (e.g. the check becomes `(false || sd_bus_track_count(subscribed) > 0)`)

Reply:
No, the sentence begins with "send to API bus", i.e. positive. "send if ... and if ..." means _only_ if ... and if ...